package/etc/conf.d/destinations/splunk_hec.conf.tmpl

```diff
@@ -25,7 +25,7 @@ destination d_hec {
             reliable(no)
             {{- end}}
 
-             disk-buf-size({{- getenv "SC4S_DEST_SPLUNK_HEC_DISKBUFF_DISKBUFSIZE" "1048576"}})
+             disk-buf-size({{- getenv "SC4S_DEST_SPLUNK_HEC_DISKBUFF_DISKBUFSIZE" "53687091200"}})
```

Comment:
this has to be math each worker thread has to divide and round nearest int

Reply:
Fix added for above.